tests/testdata/leanimports.py

```diff
@@ -5,7 +5,7 @@
 import sys
 
 # block slow imports
-for unwanted in ["deepspeed", "llama_cpp", "torch", "vllm"]:
+for unwanted in ["deepspeed", "llama_cpp", "vllm"]:
```

Comment:
Is this the only option or can we move the new torch import to a lazy load?

Reply:
the issue is coming directly from docling, so it seems like we are stuck with this change. If it were in ilab or SDG it would be an easy fix but its either coming from docling or something they import

Reply:
Would it not work to move:

```
    from instructlab.sdg.generate_data import generate_data
    from instructlab.sdg.utils import GenerateException
```

inside `gen_data` in generate_data.py?

Reply:
I figured these types of errors hinged more on the types of imports happening at lower levels. Good to know changing top level imports like this fixes these issues too! Let me give it a shot

Reply:
looks like it worked, thanks